Resources/Prototypes/Entities/Structures/Piping/Atmospherics/pipes.yml

```
@@ -7,6 +7,11 @@
   placement:
     mode: SnapgridCenter
   components:
+  # Start DeltaV additions
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/Items/crowbar_drop.ogg
+  # End DeltaV additions
```

nit: If its just one component, you can just add the comment to that component's first line.

```suggestion
  - type: EmitSoundOnLand # DeltaV
    sound:
      path: /Audio/Items/crowbar_drop.ogg
```